### ISSUE TITLE
Different statistic of FFHQ256 with precompute statistic.

### ISSUE THREAD

**Issue by guyuchao** [GaParmar/clean-fid#27]
We follow the StyleGAN-ADA to extract FFHQ256 from tfrecord file. However, when I computed the statistic, it results in different statistics from your pre-computed trainval70K. I want to know whether the calculation steps have changed?

**Comment by guyuchao** [comment]
`import numpy as np
stat_official=np.load("experiments/pretrained_models/clean_fid/ffhq_clean_trainval70k_256.npz")
print('mu_official',stat_official['mu'].sum()) # 618.9957
print('sigma_official',stat_official['sigma'].sum()) # 12467.119923723967
stat_ours=np.load("experiments/pretrained_models/clean_fid/myffhq_clean_custom_na.npz")
print('mu_our',stat_ours['mu'].sum()) # 622.30695
print('sigma_our',stat_ours['sigma'].sum()) # 12545.241799384594`

It seems the problem lies in the dataset or changed statistic calculation pipeline. I provide first ten samples of my ffhq-256 dataset(extracted from tfrecord by StyleGAN-ADA scripts) to help you compare with your ones. 
https://drive.google.com/file/d/1_8VisJJNiSDs5C9c3msiBgYB1Wl5yNWx/view?usp=sharing

If the dataset is actually the same, you may be change some pipeline in calculating statistics.

**Comment by GaParmar** [comment]
Hi, 

I looked at the images I used for my statistics and the reference 10 images you provided. 
They are indeed slightly different, which I suspect might be due to how the StyleGAN-ADA preprocess their images. 


This is the process I followed for computing my FFHQ statistics: 
-> Get the 256x256 .tfdir images by getting the file `ffhq-r08.tfrecord` from the [official location](https://drive.google.com/drive/folders/1M24jfI-Ylb-k2EGhELSnxssWi9wGUokg).
-> Extract just this .tfdir file using the extraction script provided [here](https://github.com/NVlabs/stylegan2-ada/blob/main/dataset_tool.py). 
-> The above generates a folder of 256x256 images that I use for computing my statistics.
